pkg/virtctl/sev/attestation.go

```diff
@@ -0,0 +1,290 @@
+	cmd.MarkFlagRequired(dhcertFlag)
+
+	return cmd
+}
+
+func NewQueryMeasurementCommand(clientConfig clientcmd.ClientConfig) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "query-measurement (VMI)",
+		Short: "Query the launch measurement from the SEV platform",
+		Long: `Query the launch measurement from the SEV platform
+The guest owner can verify the measurement against the measurement provided by the platform owner`,
+		Args:    templates.ExactArgs(COMMAND_QUERY_MEASUREMENT, 1),
+		Example: usage(COMMAND_QUERY_MEASUREMENT),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			c := SEVCommand{
+				command:      COMMAND_QUERY_MEASUREMENT,
+				clientConfig: clientConfig,
+			}
+			return c.Run(args)
+		},
+	}
+
+	cmd.SetUsageTemplate(templates.UsageTemplate())
+	cmd.Flags().StringVar(&outpath, outputFlag, "",
+		"Filepath where platform measurement info is to be written")
+	return cmd
+}
+
+func NewInjectSecretCommand(clientConfig clientcmd.ClientConfig) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "inject-secret (VMI)",
+		Short:   "Inject the guest owner's secret into the running VM",
+		Long:    `Inject the guest owner's secret into the running VM`,
+		Args:    templates.ExactArgs(COMMAND_INJECT_SECRET, 1),
+		Example: usage(COMMAND_INJECT_SECRET),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			c := SEVCommand{
+				command:      COMMAND_INJECT_SECRET,
+				clientConfig: clientConfig,
+			}
+			return c.Run(args)
+		},
+	}
+
+	cmd.SetUsageTemplate(templates.UsageTemplate())
+	cmd.Flags().StringVar(&secret, secretFlag, "", "(Required) Guest owner secret key")
+	cmd.MarkFlagRequired(secretFlag)
+	cmd.Flags().StringVar(&header, headerFlag, "", "(Required) Guest owner secret key header")
+	cmd.MarkFlagRequired(headerFlag)
+
+	return cmd
+}
+
+func usage(cmd string) string {
+	if cmd == COMMAND_SEV {
+		return fmt.Sprintf(" {{ProgramName}} sev <subcommand> myvmi")
+	} else {
+		return fmt.Sprintf(" {{ProgramName}} sev %s myvmi", cmd)
+	}
+}
+
+func (sc *SEVCommand) Run(args []string) error {
+	vmiName := args[0]
+	namespace, _, err := sc.clientConfig.Namespace()
+	if err != nil {
+		return err
+	}
+
+	virtClient, err := kubecli.GetKubevirtClientFromClientConfig(sc.clientConfig)
+	if err != nil {
+		return fmt.Errorf("Cannot obtain KubeVirt client: %v", err)
+	}
+
+	switch sc.command {
+	case COMMAND_SEV:
+		return fmt.Errorf("A subcommand is needed to execute the sev command")
+
+	case COMMAND_FETCH_CERT_CHAIN:
+		var file *os.File
+		pathProvided := outpath != ""
+		if pathProvided {
+			file, err = os.OpenFile(outpath, os.O_WRONLY|os.O_CREATE|os.O_EXCL, output_perm)
+			if err != nil {
+				log.Log.Errorf("[SEV] Failed to write certificate chain to %s. File already exists", outpath)
+				return fmt.Errorf("Error %s/%s: %v", namespace, vmiName, err)
+			}
+		}
+		defer file.Close()
+
+		if sevPlatformInfo, err := virtClient.VirtualMachineInstance(namespace).SEVFetchCertChain(vmiName); err != nil {
```

Comment:
I'd rewrite this as:
```golang
sevPlatformInfo, err := virtClient.VirtualMachineInstance(namespace).SEVFetchCertChain(vmiName)
if err != nil {
    return fmt.Errorf("Error %s/%s: %v", namespace, vmiName, err)
}
```
It can avoid one level indentation